templates/input.erb

```diff
@@ -10,6 +10,6 @@
   File        <%= "'#{@input_file_path}'" %>
   <%- end %>
   <%- if @input_type -%>
```

Comment:
Why do you want to remove the single quotes from the resulting config file? Just trying to make sure I understand all the changes.